tests/test_cli.py

```diff
@@ -84,6 +84,28 @@ def test_show_project_nonexistent():
     assert failed_result.exception
 
 
+def test_init_project(testdatadir):
+    dirpath = os.path.join(str(testdatadir), 'projects', 'dummy-fi')
+    fpath = os.path.join(str(dirpath), 'test_init_project_datafile')
+    os.makedirs(dirpath)
+    open(fpath, 'a').close()
+
+    assert runner.invoke(
+        annif.cli.cli,
+        ['init', 'dummy-fi']).exit_code == 0
+    assert not os.path.exists(fpath)
+
+
+def test_init_project_nonexistent(testdatadir):
+    runner.invoke(
```

Comment:
Why run this command in this test? Isn't it enough to run `init` just for `TEMP_PROJECT`?

Reply:
I think the idea was to check that `init` raises an error when a project that has entry in `projects.cfg` but no data. Removed that command now